AmazonChimeSDK/AmazonChimeSDK/internal/ingestion/database/IngestionMeetingEvent.swift

```diff
@@ -103,4 +103,31 @@ extension IngestionMeetingEvent {
         let item = eventAttributes[EventAttributeName.batteryState.description]
         return item??.value as? String
     }
+    
```

Comment:
Maybe a bit random, but where was I on convincing others to get rid of 'internal' :). Maybe if you agree we can move everything in this directory in a different PR.